en/guide/wireshark.md

```diff
@@ -0,0 +1,111 @@
+# Parsing MAVLink in Wireshark
+
+## Generate MAVLink Lua Plugin for Wireshark
+
+First you will need to generate a *Wireshark* plugin that includes definitions for the MAVLink messages that you want it to handle.
+The MAVLink generator (**mavgen**) can build this plugin for a dialect in the same way as it builds MAVLink libraries for other programming languages.
+
+The steps are:
+1. [Install MAVLink](../getting_started/installation.md) (if you have not already done so).
+1. Build libraries for your target dialect, specifing `WLua` as the target language.
+   This process is described in the toic[Generate MAVLink Libraries](../getting_started/generate_libraries.md).
+
+   For example, to build the MAVLink 2 Wireshark plugin for [common.xml](../messages/common.md) you might use the following command:
+   ```bash
+   python3 -m pymavlink.tools.mavgen --lang=WLua --wire-protocol=2.0 --output=mavlink_2_common message_definitions/v1.0/common.xml
+   ```
+   The plugin would be created in the current directory as: **mavlink_2_common.lua**. 
+
+> **Note** You will need to regenerate and reimport the plugin (as shown below) if your dialect changes.
+
+## Import Lua Plugin into WireShark
+
+The import the plugin into *Wireshark*
+
+1. Copy the plugin file into the wireshare plugins directory.
+   - On Linux systems this might be: `~/.wireshark/plugins`
+     - If you are running *Wireshark* with sudo then do:
+       - `sudo su`
+       - Copy it into `/root/.local/lib/wireshark`
+  - On Windows this might be: `Program Files/Wireshark/plugins`.
+1. The last few lines of the plugin file specify the ports to be monitored.
+   By default these are 14550 and 14580, the ports for communication with a GCS and Offboard control.
+   Change these lines and save the file if your system uses different ports.
```

Comment:
For now this is okay but you can configure Wireshark to decode traffic on other ports, no need to change the file then.

Reply:
I actually split this into a new section.